webapp/templates/jury/partials/submission_list.html.twig

```diff
@@ -191,13 +202,28 @@
                 {% else %}
                     {% set externalJudgement = submission.externalJudgements.first %}
                 {% endif %}
+                {% if external_ccs_submission_url is not empty %}
+                    {% set externalSubmissionUrl = submission | externalCcsUrl %}
+                {% else %}
+                    {% set sexternalSubmissionUrl = null %}
+                {% endif %}
                 <td>
-                    <a href="{{ link }}">
+                    <a {% if externalSubmissionUrl is not null %}href="{{ externalSubmissionUrl }}"{% endif %}>
                         External
                     </a>
                 </td>
                 <td>
-                    <a href="{{ link }}">
+                    <a {% if externalSubmissionUrl is not null %}href="{{ externalSubmissionUrl }}"{% endif %}>
+                        s{{ submission.externalid }}
```

Comment:
remove s - it adds more confusion, if external IDs are not numeric but alphanumeric